recipes/nitro/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nitro" %}
+{% set version = "2.7.dev2" %}
+{% set sha256 = "cb00e5bd5d045f1bc333e1e054272c206c9910d008698b9cd7f2d67153a32ee2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 2.7dev-2.tar.gz
```

Comment:
you might want to use the `"{{ version }}` jinja tag here and in the url to make updating this recipe easier in the future